<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="font">
         <font>
          <family>Impact</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Расчёт температуры в двухслойном материале</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>Impact</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Характеристики слоёв</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>K1 [Дж/(М*с*К)]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="K1"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>C1 [Дж/(Кг*К)]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="C1"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ρ1 [Кг/(М^3)]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="D1"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>L1 [мкМ]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="L1"/>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>T0 1 [К]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="T01"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>K2 [Дж/(М*с*К)]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="K2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>C2 [Дж/(Кг*К)]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="C2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ρ2 [Кг/(М^3)]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="D2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>L2 [мкМ]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="L2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>T0 2 [К]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="T02"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Время [мс]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Time"/>
      </item>
      <item>
       <widget class="QLabel" name="timeSteplabel">
        <property name="text">
         <string>Шаг по времени [мс]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="timeSteptau"/>
      </item>
      <item>
       <widget class="QLabel" name="coordSteplabel">
        <property name="text">
         <string>Шаг по координате [мкм]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="coordSteph"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="EnvCheckBox">
        <property name="text">
         <string>Во внешней среде (По умолчанию в вакууме)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="timeStep">
        <property name="text">
         <string>Задать шаг по времени вручную</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="coordStep">
        <property name="text">
         <string>Задать шаг по координате вручную.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="EnvSpecificName">
      <item row="0" column="1">
       <widget class="QLabel" name="EnvSpecNameLabel">
        <property name="font">
         <font>
          <family>Impact</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Характеристика среды&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>38</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="EnvSpecWarnLabel">
        <property name="text">
         <string>(без учёта вз-я на боковой поверхности)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="EnvSpecific">
      <item>
       <widget class="QLabel" name="KenvName">
        <property name="text">
         <string>K env [Дж/(М*с*К)]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Kenv"/>
      </item>
      <item>
       <widget class="QLabel" name="CenvName">
        <property name="text">
         <string>C env [Дж/(Кг*К)]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Cenv"/>
      </item>
      <item>
       <widget class="QLabel" name="DenvName">
        <property name="text">
         <string>ρ env [Кг/(М^3)]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Denv"/>
      </item>
      <item>
       <widget class="QLabel" name="T0envName">
        <property name="text">
         <string>T0 env [К]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="T0env"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Запуск</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
